airbyte-webapp/src/core/domain/connection/utils.ts

```diff
@@ -0,0 +1,24 @@
+import { WebBackendConnectionRead, WebBackendConnectionUpdate } from "core/request/AirbyteClient";
+
+export const toWebBackendConnectionUpdate = (connection: WebBackendConnectionRead): WebBackendConnectionUpdate => ({
+  name: connection.name,
+  connectionId: connection.connectionId,
+  namespaceDefinition: connection.namespaceDefinition,
+  namespaceFormat: connection.namespaceFormat,
+  prefix: connection.prefix,
+  operationIds: connection.operationIds,
+  syncCatalog: connection.syncCatalog,
+  schedule: connection.schedule,
+  status: connection.status,
+  resourceRequirements: connection.resourceRequirements,
+  operations: connection.operations,
+  sourceCatalogId: connection.catalogId,
```

Comment:
Looks like the only difference between a `WebBackendConnectionRead` and a `WebBackendConnectionUpdate` is the name of this field, `sourceCatalogId` instead of `catalogId`. Is that expected? Just making sure it isn't an oversight

Reply:
`ConnectionRead` contains a lot more fields than `update` including `operationIds`, `latestSyncJobCreatedAt`, etc. These are the ones that match with `ConnectionUpdate`